translation_layer/jpl-capi.js

```diff
@@ -0,0 +1,150 @@
+/*globals console, window*/
+define ('main',['require','jquery','ExtendedModel','api/snapshot/adapters/BackboneAdapter','sim-common/Detect','sim-common/VersionCheck','text!sim-common/templates/notSupported.html','api/snapshot/adapters/BackboneAdapter','api/snapshot/Controller','api/snapshot/Transporter','api/snapshot/SimCapiValue'],function(require){
+
+    var $ = require('jquery');
+    var $body;
+
+    var ExtendedModel = require('ExtendedModel');
+    var adapter = require('api/snapshot/adapters/BackboneAdapter').getInstance();
+
+    var Capi = ExtendedModel.extend({});
+    var capi;
+
+    require('sim-common/Detect');
+    require('sim-common/VersionCheck');
+
+    var notSupportedTemplate = require('text!sim-common/templates/notSupported.html');
+    var BackboneAdapter = require ('api/snapshot/adapters/BackboneAdapter').getInstance();
+    var Controller = require('api/snapshot/Controller');
+    var Transporter = require('api/snapshot/Transporter').getInstance();
+    var Types = require('api/snapshot/SimCapiValue').TYPES;
+
+	var initialized = false;
+
+    window.receiveExposeFromRanger = function (name, type, value, allowedValues)
```

Comment:
that is, could you alternatively have this return an object with these functions and import this file into the code that calls these functions?
Are these called from some external code, or only at the bottom of this file? It's a little confusing to me how this is consumed.